src/python/WMCore/DataStructs/File.py

```diff
@@ -105,6 +110,27 @@ def __hash__(self):
         thisHash = self['lfn'].__hash__()
         return thisHash
 
+    def __lt__(self, rhs):
```

Comment:
Nice!

Reply:
I have to admit that i had to speculate on how to sort these `DataStructs/Files`, but based on `__eq__()` checking only `'lfn'` seemded like a reasonable option